Dshop/apps/products_catalogue/models.py

```diff
@@ -158,3 +161,57 @@ class ProductAttribute(models.Model):
     product = models.ForeignKey(Product, on_delete=models.CASCADE)
     key = models.CharField(max_length=255)
     value = models.CharField(max_length=255)
+
+
+
+class DeliveryMethod(models.Model):
+    name = models.CharField(max_length=255)
+    price = models.DecimalField(max_digits=5, decimal_places=2)
+
+    def __str__(self):
+        return f'{self.name}'
+
+
+class Order(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE, default=None)
```

Comment:
```suggestion
    user = models.ForeignKey(User, on_delete=models.SET_NULL, default=None)
```
let's not delete orders.